src/python/rebalance/rebalance_rackaware.py

```diff
@@ -490,29 +486,34 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
 
             logger.debug("Start with position in Rack Alternated List: %s", str(next_Leader))
 
-            reassignment_plan, balanced_partitions_for_topic = rassignment_Generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
-            logger.debug("Already balanced partitions for the topic %s are: %s", topic, balanced_partitions_for_topic)
+            reassignment_plan, balanced_partitions_for_topic = reassignment_generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
+            logger.debug("\nAlready balanced partitions for the topic %s are: \n%s", topic, balanced_partitions_for_topic)
```

Comment:
Is "\n" at the beginning of a log message intented?

Reply:
yes it was intentional